index.html

```diff
@@ -1603,6 +1622,348 @@ <h2>
+    <li>
+      If |samples| is not an Array, return [=error=] with [=error code|WebDriver error code=] [=invalid argument=].
+    </li>
+    <li>
+      For each |sample| in |samples|:
+      <ol>
+        <li>
+          If |sample| is not a [=string=] and a valid {{PressureState}}, return [=error=] with [=error code|WebDriver error code=] [=invalid argument=].
+        </li>
+        <li>
+          [=queue/Enqueue=] |sample| to |virtualPressureSource|'s [=pending samples=].
+        </li>
+      </ol>
+    </li>
+    <li>
+      Let |topLevelTraversable| be the <a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
+      context</a>'s [=browsing context/top-level traversable=].
+    </li>
+    <li>
+      Let |topLevelVirtualPressureSourceMapping| be the |topLevelTraversable|'s [=virtual pressure source mapping=].
+    </li>
+    <li>
+      If |topLevelVirtualPressureSourceMapping| contains |virtualPressureSourceType|, return [=error=] with
+      [=error code|WebDriver error code=] [=invalid argument=].
+    </li>
+    <li>
+      Let |virtualPressureSource| be a new [=virtual pressure source=].
+    </li>
+    <li>
+      Let |virtualPressureSource|'s [=sample update interval=] be |sampleInterval|.
+    </li>
+    <li>
+      Run the following steps [=in parallel=] at the |sampleInterval| time interval:
+      <ol>
+        <li>
+          Remove |virtualPressureSource|'s [=pending samples=] first item if the queue is not empty.
+        </li>
+      </ol>
+    </li>
+    <li>
+      Set |topLevelVirtualPressureSourceMapping|[|virtualPressureSourceType|] to |virtualPressureSource|.
+    </li>
+    <li>
+      Return [=success=] with data <code>null</code>.
+    </li>
+  </ol>
+  <h4>
+    Delete virtual pressure source
+  </h4>
+  <table class="def">
+    <tr>
+      <th>
+        HTTP Method
+      </th>
+      <th>
+        [=extension command URI Template|URI Template=]
+      </th>
+    </tr>
+    <tr>
+      <td>
+        DELETE
+      </td>
+      <td>
+        /session/{session id}/pressuresource/{type}
+      </td>
+    </tr>
+  </table>
+  <p>
+    This [=extension command=] deletes a given [=source type=] of [=virtual pressure source=] and returns
+    pressure source updates back to hardware.
+  </p>
+  <p>
+    The [=remote end steps=] given |session|, |URL variables| and |parameters| are:
+  </p>
+  <ol class="algorithm" data-cite="WEBDRIVER2">
+    <li>
+      Let |virtualPressureSourceType| be the value of the |URL variables|["type"].
+    </li>
+    <li>
+      Let |topLevelTraversable| be the <a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
+      context</a>'s [=browsing context/top-level traversable=].
+    </li>
+    <li>
+      Let |topLevelVirtualPressureSourceMapping| be the |topLevelTraversable|'s [=virtual pressure source mapping=].
+    </li>
+    <li>
+      Remove |topLevelVirtualPressureSourceMapping|[|virtualPressureSourceType|] and any data associated with its
```

Comment:
What does "any data associated with it" mean?

Reply:
virtualPressureSourceType has values attached, like internal slots. Maybe I should say delete virtualPressureSourceType, or do you have a better wording?

Reply:
My understanding is that [removing](https://infra.spec.whatwg.org/#map-remove) an entry from the map would also get rid of whatever was stored there. If not, it is better to be explicit and mention what steps need to be carried out.